File: Daily Test/Java/Immediate Smaller To M 4.5.2026.java
import java.util.*;

public class Hello {
    public static void main(String[] args) {
        Scanner s = new Scanner(System.in);

        int n = s.nextInt();
        int[] arr = new int[n];

        for(int i = 0; i < n; i++)
            arr[i] = s.nextInt();

        int m = s.nextInt();
        int ans = -1;

        for(int x : arr) {
            if(x < m && x > ans) {
                ans = x;
            }
        }

        System.out.print(ans);
    }
}

File: Daily Test/C/Immediate Smaller To M 4.5.2026.c
#include <stdio.h>

int main() {
    int n, m;
    scanf("%d", &n);

    int arr[n];
    for(int i = 0; i < n; i++)
        scanf("%d", &arr[i]);

    scanf("%d", &m);

    int ans = -1;
    for(int i = 0; i < n; i++) {
        if(arr[i] < m && arr[i] > ans) {
            ans = arr[i];
        }
    }

    printf("%d", ans);
    return 0;
}